pkg/cmd/stack/translate.go

```diff
@@ -58,6 +60,9 @@ func translateEnvVars(s *model.Stack) error {
 		for _, envFilepath := range svc.EnvFiles {
 			translateEnvFile(&svc, envFilepath)
 		}
+		sort.SliceStable(svc.Environment, func(i, j int) bool {
```

Comment:
why do we need to do this? To make it easier to read?

Reply:
Kubernetes redeploys pods if envvars have a different order. This is very inconvenient with stefulsets, for example. If my stack yaml does change, it shouldn't be expected that pods are redeployed